studybuddies/src/app/components/student/view-class-student/view-class-student.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ViewClassStudentComponent } from './view-class-student.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7173073)
```suggestion
import { ViewClassStudentComponent } from "./view-class-student.component";
```